backends-velox/src/main/scala/io/glutenproject/utils/VeloxExpressionUtil.scala

```diff
@@ -31,7 +31,6 @@ object VeloxExpressionUtil {
   // The expression with empty type will fall back directly.
   final val VELOX_EXPR_BLACKLIST: Map[String, Set[String]] = Map(
     CAST -> Set(ARRAY_TYPE, MAP_TYPE, STRUCT_TYPE),
-    ROUND -> Set(EMPTY_TYPE),
```

Comment:
Is there any UT to verify the round() with Vanilla Spark results for Velox backend?

Reply:
Added a unit test. I also find there are some tests for round in `GlutenMathFunctionsSuite`, will test on that after your PR being merged.